assets/image-manifest.yaml

```diff
@@ -382,6 +382,38 @@ EmrReleases:
       User: hadoop:hadoop
       WorkingDir: /home/hadoop
   ReleaseName: emr-6.13.0
+- !EmrRelease
+  Images:
+  - !ImageDetail
+    EnvironmentVariable:
+    - java_home
+    - spark_home
+    FileStructure:
+    - spark-jars-6-12
```

Comment:
nit: it would be more readable to sort this list.

Reply:
Thanks, this is auto-generated by scripts, I can try to sort it next time.